tests/test_math/test_lattice.py

```diff
@@ -17,10 +17,23 @@
 import numpy as np
 
 from mrmustard.lab import Gaussian
+from mrmustard import settings
 
 
-def test_vanilla_vs_binomial():
-    """Test that the vanilla and binomial methods give the same result"""
+def test_vanillaNumba_vs_binomial():
+    """Test that the vanilla method (Numba with precision of complex128) and the binomial method give the same result"""
+    settings.PRECISION_BITS_HERMITE_POLY = 128
+    G = Gaussian(2)
+
+    ket_vanilla = G.ket(cutoffs=[10, 10])[:5, :5]
+    ket_binomial = G.ket(max_photons=10)[:5, :5]
+
+    assert np.allclose(ket_vanilla, ket_binomial)
+
```

Comment:
Set precision back: 

At the beginning of the file (or at the beginning of the func), add:
`p0 = settings.PRECISION_BITS_HERMITE_POLY`

At the beginning of the function:
` settings.PRECISION_BITS_HERMITE_POLY = precision`

At the end of the function, add:
` settings.PRECISION_BITS_HERMITE_POLY = p0`